backend/djangoindia/api/serializers/volunteer.py

```diff
@@ -0,0 +1,13 @@
+from db.models.volunteers import EventVolunteers
+from rest_framework import serializers
+
+class EventVolunteerSerializer(serializers.Serializer):
+    full_name = serializers.CharField(max_length=100)
+    email = serializers.EmailField()
+    profile_pic = serializers.ImageField(upload_to='') # Don't know the location so kept blank
+    linkedin_url = serializers.URLField()
+    twitter_url = serializers.URLField()
```

Comment:
do we still call twitter or x? :p
@DevilsAutumn

Reply:
I have designed the model by the field names they have given

Reply:
@ankanchanda lets keep twitter only, x doesn't sound good XD

Reply:
elon named x for the same reason